core/Middlewares/DetectModuleMiddleware.php

```diff
@@ -4,12 +4,17 @@
 
 use icms;
 use icms_module_Object;
-use ImpressCMS\Core\Controllers\LegacyController;
+use ImpressCMS\Core\Models\Module;
 use League\Route\Route;
```

Comment:
still using league route here, I thought this was replaced now.

Reply:
And here fixed too!